app/src/androidTest/java/com/nervousfish/nervousfish/test/MainSteps.java

```diff
@@ -170,11 +180,6 @@ public void iShouldGoToSettingsScreen() {
         intended(hasComponent(SettingsActivity.class.getName()));
     }
 
-    @Then("^the app shouldn't crash because of Bluetooth$")
```

Comment:
Because the emulators don't have bluetooth, and on some way the button showed up like it does have Bluetooth. This gave crashes and made me unfortunately delete the test